### PR TITLE
feature(enterprise): optional auto stock accounting for the enterprise

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR add the feature for the enterprise to decide to use auto stock accounting (write automatically transactions in the journal for stock movements) or not.

### CODE REVIEW

File: client/src/i18n/en/enterprise.json [hunk 1]
```diff
@@ -21,7 +21,9 @@
       "ENABLE_PREPAYMENT_INVOICE_LABEL" : "Enable to see the balance on invoice",
       "ENABLE_PREPAYMENT_INVOICE_HELP_TEXT" : "Enabling this feature allows to see the balance and the paid amount on an invoice",
       "ENABLE_BARCODES_LABEL" : "Enables barcodes throughout the application",
-      "ENABLE_BARCODES_HELP_TEXT" : "Enabling this feature will place barcodes on most printed records and enable options to scan barcodes for document inputs."
+      "ENABLE_BARCODES_HELP_TEXT" : "Enabling this feature will place barcodes on most printed records and enable options to scan barcodes for document inputs.",
+      "ENABLE_AUTO_STOCK_ACCOUNTING_LABEL" : "Enables realtime stock accounting",
+      "ENABLE_AUTO_STOCK_ACCOUNTING_HELP_TEXT" : "Enabling this feature will write into the posting journal in realtime all stock movements transaction. It require that all inventory accounts must be correctly configured"
```
Comment (jniles) [inline client/src/i18n/en/enterprise.json]: This is a really good translation!  I'm impressed :)

I'm not completely sure why, but in English it makes more sense to reverse your phrases a little bit:
> Enabling this feature will write stock movement transactions into the posting journal in real time.  It requires that all inventory accounts are correctly configured.

File: client/src/modules/enterprises/enterprises.js [hunk 2]
```diff
@@ -76,9 +76,9 @@ function EnterpriseController(Enterprises, util, Notify, Projects, Modal, Scroll
 
     changes.settings = angular.copy(vm.enterprise.settings);
 
-    const promise = (creation) ?
-      Enterprises.create(changes) :
-      Enterprises.update(vm.enterprise.id, changes);
+    const promise = (creation)
+      ? Enterprises.create(changes)
```
Comment (jniles) [inline client/src/modules/enterprises/enterprises.js:80]: Nice 👍

**Review by jniles — COMMENTED** [review]
Logic looks right to me.  I'll need to download it and run it to make sure it behaves correctly, but on first glance it looks really good!  Well done!

**Comment by mbayopanda** [comment]
@jniles

**Comment by jniles** [comment]
LGTM.  Thanks!

bors r+

**Comment by jniles** [comment]
Looks like a fluke.

Bors r+

**Pull request merged**